kie-wb-common-react/boxed-expression-editor/src/@types/react-table.d.ts

```diff
@@ -60,9 +62,13 @@ declare module "react-table" {
     getResizerProps: (props?: Partial<TableResizerProps>) => TableResizerProps;
     /** It tells whether column can resize or not */
     canResize: boolean;
+    /** It tells whether column is of type counter or not */
+    isCountColumn: boolean;
+    /** Disabling table handler on the header of this column */
+    disableHandlerOnHeader?: boolean;
   }
 
   export interface DataRecord {
-    [columnName: string]: string;
+    [columnName: string]: unknown;
```

Comment:
My point of view is that since `DataRecord` is part of `react-table.d.ts` types extensions, it should be as more agnostic as possible about the values that a given cell can hold.
In our cases a cell can be a `string` in the case of `Relation` table, and a more complex type in case of `Context` table.
When we will cover `DecisionTable` maybe will be different and so on. So using a top type as `unknown` in such situation is OK.